### ISSUE TITLE
Accessibility and User Experience

### ISSUE THREAD

**Issue by qisforq** [AthletiFi/athletifi-website#40]
- Issue: Potential accessibility issues (e.g., non-descriptive alt texts for images, lack of ARIA labels).

- Suggestion: Audit and improve accessibility, ensure keyboard navigability, and use ARIA attributes.

**Comment by chef-louis** [comment]
A pretty good resource ([link](https://www.washington.edu/accesscomputing/30-web-accessibility-tips)) for considering what accessibility areas we want to include into the application

**Comment by chef-louis** [comment]
Came up with a game plan with @pm-moyanor after reviewing the list of accessibility features.

- [ ] Ensure all images have proper alt text
- [ ] Use headings properly to properly form an outline of the page (and not skip heading levels)
- [ ] Use ARIA landmarks
- [ ] Use labels for form fields to allow screen reader users to associate labels with input fields correctly
- [ ] Group related form fields together (ie. apply `fieldset` and `legend` elements where appropriate)
- [ ] Test web pages using only keyboard (ie. #nomouse challenge)
- [ ] Test pages on mobile device

**Comment by chef-louis** [comment]
@pm-moyanor I've taken a pass at the first 2 sub-tasks and pushed change to a local branch [here](https://github.com/chef-louis/athletifi-website/tree/accessibility). I've got some other things to take care of this afternoon but let me know if you wanted to tackle any of these specifically and I'll work on the others.

**Comment by pm-moyanor** [comment]
great @chef-louis I can grab anything, only I'll have time to get to it on Monday

**Comment by chef-louis** [comment]
@pm-moyanor No worries -- I'll keep going and see how far I get today.

**Comment by chef-louis** [comment]
Okay ran through a "no-mouse" test and checked out the application on the mobile device. There are some design aspects that can be better (padding, margin, etc) for the mobile layout, but functionality and accessibility appear to be okay from what I can tell.
